internal/http/apitester_test.go

```diff
@@ -91,10 +96,11 @@ func TestAPITester_GetEventResult(t *testing.T) {
 	defer ts.Close()
 
 	c := &APITester{
-		HTTPClient: ts.Client(),
-		URL:        ts.URL,
-		Username:   "dummy",
-		AccessKey:  "accesskey",
+		HTTPClient:         NewRetryableClient(TestClientTimedOut),
```

Comment:
This makes the test `Buggy Event` run for +7s:
```
=== RUN   TestAPITester_GetEventResult/Buggy_Event
    --- PASS: TestAPITester_GetEventResult/Buggy_Event (7.01s)
```

That's because the default configuration is intended for production, not testing.
E.g. those values are not suitable for an integration test.
```
		RetryWaitMin: 1 * time.Second,
		RetryWaitMax: 30 * time.Second,
		RetryMax:     3,
```

Reply:
updated.